interface/ceed-elemrestriction.c

```diff
@@ -308,6 +308,54 @@ int CeedElemRestrictionApply(CeedElemRestriction rstr, CeedTransposeMode tmode,
   return 0;
 }
 
+/**
+  @brief Restrict an L-vector to a block of an E-vector or apply transpose
+
+  @param rstr    CeedElemRestriction
+  @param block   Block number to restrict to/from
+  @param tmode   Apply restriction or transpose
+  @param lmode   Ordering of the ncomp components
+  @param u       Input vector (of size @a ndof when tmode=CEED_NOTRANSPOSE)
+  @param v       Output vector (of size @a nelem * @a elemsize when tmode=CEED_NOTRANSPOSE)
```

Comment:
These numbers aren't directly defined here. An example might be useful.

Reply:
Updated:

```
  @param block   Block number to restrict to/from, i.e. block=0 will handle
                 elements [0 : blksize-1] and block=3 will handle elements
                 [3*blksize : 3*blksize-1]
```